index.html

```diff
@@ -59,6 +60,214 @@ <h3 class="connect">LET'S CONNECT</h3>
             </li>
             
           </ul>
+        </section>
+
+            <!--Card One-->
+        <section class="section-1">
+
+          <div class="first-card " >
+            <div class="items">
+              <div>
+                <img src="./images/Snapshoot Portfolio.png" alt="">
+              </div>
+
+              <div>
+                <h1>Tonic</h1>
+              </div>
+
+              <div class="details">
+                <h3>CANOPY</h3>
+                <img src="./images/Counter.png" alt="">
+                <h4>Back End Dev</h4>
+                <img src="./images/Counter.png" alt="">
+                <h4>2015</h4>
+              </div>
+
+              <p>
+                A daily selection of privately<br>personalised reads; no accounts or<br>sign-ups required.
+              </p>
+
+              <ul class="languages">
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>JavaScript</li>
+              </ul>
+          
+              <button class="see-project"><a href="#">
+                See Project</a>
+              </button>
+            </div>
```

Comment:
- [x] Let's focus on this card as the case study. I am not able to access some of the stylings since there was an inheritance of colors from other sections.  Kindly stick to the Figma design as much as possible. This is particularly on the colors. The colors and border button colors mostly used on this card, include `#172B4D`, `#344563`, `#7A869A`. Please modify the colors to match the Figma design. This will make our project more professional.
- [ ] In addition, ensure that the `font-size` of the texts match the Figma design.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/228761415-496efae2-c015-473b-a918-965f814e3b56.png)


Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/228258239-512b7079-d47f-46aa-8af9-8a3ef9316bcd.png)